### PR TITLE
Update docs for conversion.register().

### LINKED ISSUE #1629: Wrong docs for conversion.register()
There's some leftovers from #1613 PR:

https://github.com/ckeditor/ckeditor5-engine/blob/a49fd54dbbe917746d1065ec36b172f1c4d7c876/src/conversion/conversion.js#L73-L82

### PR DESCRIPTION
**Pull request by jodator**
### Suggested merge commit message ([convention](https://github.com/ckeditor/ckeditor5-design/wiki/Git-commit-message-convention))

Docs: Update docs for conversion.register(). Closes #1629.

---

### Additional information

* Somehow we missed it at the review.

### CODE REVIEW

File: src/conversion/conversion.js [hunk 1]
```diff
@@ -70,17 +70,12 @@ export default class Conversion {
...
 	 * {@link module:utils/ckeditorerror~CKEditorError `conversion-register-group-exists` error} is thrown.
 	 *
-	 * @param {Object} options
-	 * @param {String} options.name The name for dispatchers group.
-	 * @param {module:engine/conversion/downcastdispatcher~DowncastDispatcher|
-	 * module:engine/conversion/upcastdispatcher~UpcastDispatcher|Array.<module:engine/conversion/downcastdispatcher~DowncastDispatcher|
-	 * module:engine/conversion/upcastdispatcher~UpcastDispatcher>} options.dispatcher Dispatcher or array of dispatchers to register
-	 * under the given name.
+	 * @param {String} name The name for dispatchers group.
 	 * @param {module:engine/conversion/downcasthelpers~DowncastHelpers|
```
Comment (pjasiun) [inline src/conversion/conversion.js]: Why not just `ConversionHelpers`?

Comment (pjasiun) [reply src/conversion/conversion.js]: It is the base of both `DowncastHelpers` and `DowncastHelpers`.

Comment (jodator) [reply src/conversion/conversion.js]: I liked the idea of directly linking here to the most used cleses.

So one click less.

Comment (pjasiun) [reply src/conversion/conversion.js]: Ok, then :)

**Pull request merged**